tests/testthat/test_updateMeta.R

```diff
@@ -58,3 +62,10 @@ test_that("illegal variable names", {
   expect_equal(mess[2], "Alter has been renamed to AlterVar\n")
   expect_equal(names(out_both$dat), c("ID1", "V1", "AlterVar"))
 })
+
+test_that("updateMeta extractData combination", {
+  suppressWarnings(newDat_ill <- extractData2(dfSAV, labels2character = namesGADS(dfSAV), dropPartialLabels = FALSE))
+  suppressMessages(out_both <- updateMeta(dfSAV, newDat_ill))
+  expect_equal(attr(out_both$dat[[1]], "label"), NULL)
+  expect_equal(attr(out_both$dat[[3]], "label"), NULL)
+})

```

Comment:
Looks all good to me and works fine. (However, I checked this only superficially and forgot about what exactly `extractData2()` [and `updateMeta()`] do in detail here 🙈)